projects/components/src/quick-search/quick-search-result.ts

```diff
@@ -7,6 +7,31 @@
  * The interface a spotlight search result item should implement
  */
 export interface QuickSearchResult {
+    /**
```

Comment:
I don't understand why we need two interfaces with a very similar comment

> The interface a quick search result item should implement. 

It seems like the first one should be `QuickSearchResults` since it's a list of results and the second one `QuickSearchResultItem` should be private, users of this class should only use `QuickSearchResultsType` (adding the s since it's a list)